src/overmap.cpp

```diff
@@ -3604,6 +3646,100 @@ std::vector<point_abs_omt> overmap::find_extras( const int z, const std::string
     return extra_locations;
 }
 
+bool overmap::passable( const tripoint_om_ms &p )
+{
+    point_om_omt omt_origin;
+    tripoint_omt_ms index;
+    std::tie( omt_origin, index ) = project_remain<coords::omt>( p );
+    std::shared_ptr<map_data_summary> &ptr = layer[index.z() +
+            OVERMAP_DEPTH].map_cache[omt_origin];
+    if( !ptr ) {
+        // Oh no we aren't populated???
+        // Promote to error later.
+        return false;
+    }
+    return ptr->passable[index.y() * 24 + index.x()];
+}
+
+void overmap::set_passable( const tripoint_om_ms &p, bool new_passable )
+{
+    point_om_omt omt_origin;
+    tripoint_omt_ms index;
+    std::tie( omt_origin, index ) = project_remain<coords::omt>( p );
+    std::shared_ptr<map_data_summary> &ptr = layer[index.z() +
+            OVERMAP_DEPTH].map_cache[omt_origin];
+    if( !ptr ) {
+        // Oh no we aren't populated???
+        // Promote to error later.
+        return;
+    }
+    if( ptr->placeholder ) {
+        // Copy the placeholder data.
+        ptr = std::make_shared<map_data_summary>( ptr.get() );
```

Comment:
If you only want to copy the placeholder then use the constructor you already had for that. But I think you mean to make a deep copy here?
```suggestion
        ptr = std::make_shared<map_data_summary>( ptr->placeholder );
```

Reply:
No just the data,  and yea that's so much cleaner.

Reply:
Edited the code suggestion to be, um, correct.